java-checks/src/test/files/checks/LambdaTooBigCheck.java

```diff
@@ -0,0 +1,26 @@
+class A {
+
+  Callable<Integer> c1 = () -> { // Noncompliant [[sc=26;ec=31;secondary=13]] {{Reduce this lambda expression number of lines from 11 to at most 10.}}
```

Comment:
I know you copy-pasted this code from old implementation, but this does not compile.
Please use the adequate import, or define a `Callable` class to be used in such context.